src/rgw/rgw_lc.cc

```diff
@@ -1055,6 +1067,7 @@ class LCOpAction_NonCurrentExpiration : public LCOpAction {
       return false;
     }
 
+    /* XXX Ok! called from LCOpRule::build() */
```

Comment:
oops--misread this code on my 2nd pass through--mtime needs to be lifted out too